annotations_levels.js

```diff
@@ -38,20 +38,23 @@ function getAnnotationLevel (severity, confidence) {
         case 'HIGH' : result = annotationsLevels.severityHIGHconfidenceHIGH; break
         case 'MEDIUM' : result = annotationsLevels.severityHIGHconfidenceMEDIUM; break
         case 'LOW' : result = annotationsLevels.severityHIGHconfidenceLOW; break
+        default: result = annotationsLevels.severityHIGHconfidenceHIGH
```

Comment:
Since we're not using these changes for this PR, I would ask you put this in a separate PR.

Reply:
I just realized that we are giving a default annotation level when we initialize the result value in the begging of the function. The default annotation level when there is no confidence provided is "warning" whatever the severity is.

Do we want to give other default annotation level based on the severity when there is no confidence provided?

Reply:
I will remove the default statements from this switch for now.